annotationPipeline/src/main/java/org/cbioportal/annotation/pipeline/MutationRecordReader.java

```diff
@@ -92,11 +82,30 @@
 
     @Override
     public void open(ExecutionContext ec) throws ItemStreamException {
-
+        this.summaryStatistics = new AnnotationSummaryStatistics(annotator);
         String genomeNexusVersion = annotator.getVersion();
 
         processComments(ec, genomeNexusVersion);
+        List<MutationRecord> mutationRecords = loadMutationRecordsFromMaf();
+        if (postIntervalSize > 0) {
+            try {
+                this.allAnnotatedRecords = annotateRecordsWithPOST(mutationRecords);
+            } catch (Exception ex) {
+                LOG.error("ERROR ANNOTATING WITH POST REQUESTS");
+                throw new RuntimeException(ex);
```

Comment:
I guess here we throw an exception because we are losing lots of records at once.  In GET, below, we don't throw an exception but rather log the error.  Are we sure we want to handle them differently?  I think probably yes.